### DISCUSSION TITLE
How to initiate keyboard ENTER event if a text field can only be successfully existed after doing that?

### DISCUSSION THREAD

**Discussion by ideepakmathur** [AutomaApp/automa#491] [Q&A]
Hi @Kholid060,

This is amazing tool and I love your idea and the beautiful representation and automation.

I was trying to test this on some site, where I need to post URL by clicking a button then I need to click enter to submit that URL but I'm unable to do that. Can you please guide me how can I achieve this or please add this in your next to do list.

Thank you

**Comment by Kholid060** [comment]
Hi,

Unfortunately, it's currently not possible to emulate the Enter key

**Reply by 9kopb** [reply]
> Hi,
> 
> Unfortunately, it's currently not possible to emulate the Enter key

Thanks for update =]
![image](https://user-images.githubusercontent.com/140489/168998697-e3801514-84ad-451d-890f-057e37aa29e3.png)
